frontend/include/chpl/framework/Context.h

```diff
@@ -43,7 +43,12 @@
 namespace chpl {
   class Context;
 
-class ErrorBase;
+  // Whether or not to exit on assertion failure
+  extern bool assertionsAreFatal;
+  // Whether or not to enable assertions
+  extern bool assertionsAreOn;
```

Comment:
I'd expect these to be declared in assertions.h / defined in assertions.cpp as well as the `set` calls for them

Reply:
I also think if we have setters we should also have getters and not expose that these variables in the header as well